homework_9/hw9.py

```diff
@@ -0,0 +1,248 @@
+#  Задание1
+
+import datetime
+
+class Chat:
+    
+    def __init__(self, chat_history=[]):
+        self.chat_history = chat_history
+    
+    def show_last_message(self):
+        return self.chat_history[0]
+    
+    def show_chat(self):
+        for sms in self.chat_history[::-1]:
+            print(sms, end='\n')
+    
+    def recieve(self, sms):
+        self.chat_history.insert(0, sms)
+
+
+class Message:
+    
+    def __init__(self, text, user):
+        
+        self.text = text
+        self.datetime = datetime.datetime.now()
```

Comment:
Этот атрибут должен задаваться только в момент отправки в чат, а то этого момента можно сделать его `None`
```suggestion
        self.datetime = None
```